docker-api/pom.xml

```diff
@@ -122,6 +122,29 @@
                 <artifactId>bundle-plugin</artifactId>
                 <extensions>true</extensions>
             </plugin>
+            <plugin>
```

Comment:
why is this needed?

Reply:
To allow building `zone-application` without downloading snapshot releases from artifactory :)

Reply:
So mbuild builds something in addition to other than mvn, or renames files?